jenkins/Jenkinsfile

```diff
@@ -264,32 +265,43 @@ pipeline {
         when { expression { !shouldAbortBuild } }
         steps {
           sh """
-            git submodule init && git submodule update
-            cd jenkins/dataproc-custom-images/dataproc-custom-images
-            python generate_custom_image.py \
-              --image-name "$IMAGE_BASE_NAME-$imageID" \
-              --dataproc-version "1.2.79-debian9" \
-              --customization-script ../prepare-custom-leonardo-jupyter-dataproc-image.sh \
-              --zone us-central1-a \
-              --gcs-bucket $BUCKET_NAME \
-              --project-id=$GOOGLE_PROJECT \
-              --disk-size=30
+              git submodule init && git submodule update
           """
+          script {
+            for(String dp_version: dataproc_versions.trim().split(' ')) {
+              def dp_version_formatted = dp_version.replace(".","-")
```

Comment:
what does the transforming for? 🤔

Reply:
We can't put periods in the image name, or else you get this: 
```ERROR: (gcloud.compute.disks.create) Could not fetch resource:
 - Invalid value for field 'resource.name': 'custom-leo-image-1.2.79-debian9-eafb4cb-install'. Must be a match of regex '(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)'```